server/service/hosts_test.go

```diff
@@ -2417,63 +2416,6 @@ func TestSetDiskEncryptionNotifications(t *testing.T) {
 			},
 			expectedError: false,
 		},
-		{
```

Comment:
This was removed because these test validate the EnableBitLockerPINProtectorConfig behavior (which was removed)